The error is:

```
While building module 'Clang_Index' imported from /jenkins/workspace/apple-clang-stage2-configure-RA_master/llvm-project/clang/lib/CrossTU/CrossTranslationUnit.cpp:20:
While building module 'Clang_Index_C' imported from /jenkins/workspace/apple-clang-stage2-configure-RA_master/llvm-project/clang/include/clang/Index/IndexDataStoreSymbolUtils.h:13:
In file included from <module-includes>:1:
/jenkins/workspace/apple-clang-stage2-configure-RA_master/llvm-project/clang/include/indexstore/indexstore.h:290:9: error: non-defining declaration of enumeration with a fixed underlying type is only permitted as a standalone declaration; missing list of enumerators? [-Welaborated-enum-base]
typedef INDEXSTORE_OPTIONS(uint64_t, indexstore_symbol_property_t) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/jenkins/workspace/apple-clang-stage2-configure-RA_master/llvm-project/clang/include/indexstore/indexstore.h:103:79: note: expanded from macro 'INDEXSTORE_OPTIONS'
# define INDEXSTORE_OPTIONS(_type, _name) enum INDEXSTORE_OPTIONS_ATTRS _name : _type _name; enum INDEXSTORE_OPTIONS_ATTRS _name : _type
                                                                              ^~~~~~~
```

--- a/clang/include/indexstore/indexstore.h
+++ b/clang/include/indexstore/indexstore.h
@@ -99,7 +99,7 @@
 #define INDEXSTORE_OPTIONS_ATTRS INDEXSTORE_OPEN_ENUM_ATTR INDEXSTORE_FLAG_ENUM_ATTR
 
 #if defined(__has_extension)
-#if __has_extension(cxx_strong_enums) || __has_feature(objc_fixed_enum)
+#if __has_feature(objc_fixed_enum)
 # define INDEXSTORE_OPTIONS(_type, _name) enum INDEXSTORE_OPTIONS_ATTRS _name : _type _name; enum INDEXSTORE_OPTIONS_ATTRS _name : _type
 #endif
 #endif

--- a/clang/include/indexstore/indexstore.h
+++ b/clang/include/indexstore/indexstore.h
@@ -98,6 +98,12 @@
 
 #define INDEXSTORE_OPTIONS_ATTRS INDEXSTORE_OPEN_ENUM_ATTR INDEXSTORE_FLAG_ENUM_ATTR
 
+#if defined(__has_extension)
+#if __has_extension(cxx_strong_enums) || __has_feature(objc_fixed_enum)
+# define INDEXSTORE_OPTIONS(_type, _name) enum INDEXSTORE_OPTIONS_ATTRS _name : _type _name; enum INDEXSTORE_OPTIONS_ATTRS _name : _type
+#endif
+#endif
+
 #ifndef INDEXSTORE_OPTIONS
 # define INDEXSTORE_OPTIONS(_type, _name) _type _name; enum INDEXSTORE_OPTIONS_ATTRS
 #endif

